docusaurus/docs/learn/quickstarts/services/kubernetes-service.md

```diff
@@ -2,69 +2,81 @@
 title: Kubernetes Service
 ---
 
-This is a quick example for tunneling to a Kubernetes workload with OpenZiti that builds on [the local Kubernetes quickstart](/learn/quickstarts/network/local-kubernetes.md). There are two deployments for you to deploy in the same namespace:
+This is a tutorial for tunneling a Kubernetes workload with OpenZiti that creates:
 
-* a demo workload service `hello-toy`
+* a demo service `hello-toy`
 * an OpenZiti reverse proxy pod `ziti-host`
 
-1. Create OpenZiti configs, service, and policies for the Hello demo deployment.
+## Steps
+
+1. You must have a OpenZiti router in the cluster with a tunnel binding. You can use this command to find eligible routers.
 
     ```text
-    ziti edge create identity device "hello-host" \
-        --jwt-output-file /tmp/hello-host.jwt --role-attributes hello-hosts
+    ziti edge list edge-routers 'isTunnelerEnabled=true'
+    ```
 
-    ziti edge enroll /tmp/hello-host.jwt
+    If none of the routers shown are in your cluster, then refer to [the router deployment guide](/guides/deployments/30-kubernetes/kubernetes-router.mdx) for more information on how to deploy a router.
 
-    ziti edge create config "hello-intercept-config" intercept.v1 \
-        '{"protocols":["tcp"],"addresses":["minihello.ziti"], "portRanges":[{"low":80, "high":80}]}'
+1. Add a role to the router's tunneler identity. This labels the identity so we can give it permission to host the service as a reverse-proxy. Substitute the router name for `router1`.
 
-    ziti edge create config "hello-host-config" host.v1 \
-        '{"protocol":"tcp", "address":"minihello.hello-toy.svc","port":80}'
+    ```text
+    ziti edge update identity "router1" \
+        --role-attributes hello-hosts
+    ```
 
-    ziti edge create service "hello-service" \
-        --configs hello-intercept-config,hello-host-config
+1. Enroll an identity for yourself. Add the identity to any tunneler so you can access `http://hello.ziti.internal` when you finish the steps.
 
-    ziti edge create service-policy "hello-bind-policy" Bind \
-        --service-roles '@hello-service' --identity-roles '#hello-hosts'
+    ```text
+    ziti edge create identity "hello-client" \
+        --role-attributes hello-clients \
+        --jwt-output-file hello-client.jwt
+    ```
 
-    ziti edge create service-policy "hello-dial-policy" Dial \
-        --service-roles '@hello-service' --identity-roles '#hello-clients'
+1. Ensure you have router policies for your identities and services.
```

Comment:
i feel like this step is "just noise". i can understand why you put it here but i'd probably remove it. that or maybe you could move this step to the end and instead, just use `policy-advisor identities -q` ?

Reply:
https://github.com/openziti/ziti-doc/pull/874